src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.ts

```diff
@@ -59,6 +59,7 @@ export class ProgrammingExerciseUpdateComponent implements OnInit {
     existingCategories: ExerciseCategory[];
 
     public inProductionEnvironment: boolean;
+    checkedFlagForStructuredGradingInstructions = false;
```

Comment:
For editing of programming exercises: Is is possible to check whether grading instructions have been created and then set this flag accordingly?

Reply:
Nevermind, it seems like this was conscious decision when the structured grading instructions were created